steward/src/commands/cork_proposal/cellar_v1/initiate_shutdown.rs

```diff
@@ -0,0 +1,55 @@
+use crate::{application::APP, cellars, commands::cork_proposal::print_proposal, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use steward_proto::steward::{
+    cellar_v1_governance::{Function, InitiateShutdown},
+    governance_call::Call,
+    CellarV1Governance, GovernanceCall,
+};
+
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nCalls initiateShutdown() on the target cellar contract at the specified block height.\nFor more information see https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol"
```

Comment:
`cellar-contracts` is going to be pointing at Cellar V2. You probably want https://github.com/PeggyJV/cellar-v1_5/blob/release/src/base/Cellar.sol